packages/@o3r/schematics/package.json

```diff
@@ -55,6 +54,7 @@
     "@nrwl/js": "~15.9.0",
     "@nrwl/linter": "~15.9.0",
     "@o3r/build-helpers": "workspace:^",
+    "@o3r/dev-tools": "workspace:^",
```

Comment:
shouldn't this PR remove the dependency?

Reply:
It is now a devDep, used only in test purpose